hillel_django/schema.py

```diff
@@ -93,9 +103,89 @@ class Meta:
         serializer_class = CategorySerializer
 
 
+class OrderProductMutation(graphene.Mutation):
+
+    class Arguments:
+        order_id = graphene.Int(required=True)
+        product_id =graphene.Int(required=True)
+        quantity = graphene.Int(required=True)
+
+    order_products = graphene.Field(OrderProductType)
+
+    @classmethod
+    def mutate(cls, root, info, order_id, product_id, quantity):
+        order_products = OrderProduct.objects.create(
+            order = Order.objects.get(id=order_id),
+            product = Product.objects.get(id=product_id),
```

Comment:
can you check of product exists (and order as well, and raise meaningful errors, in case they are not exist)?

Reply:
Database updates with new order and orderproducts, when proucts exists, and nothing happens otherwise. But in GraphiQL it returns None, and user doesn`t see what he do wrong (if i sent product or user that not exists). For now I have not find decision for this problem, but I`ll figure it out in the near future.